crates/solver/src/settlement_submission/submitter.rs

```diff
@@ -53,6 +53,9 @@ pub enum SubmitApiError {
     InvalidNonce,
     ReplacementTransactionUnderpriced,
     OpenEthereumTooCheapToReplace, // todo ds safe to remove after dropping OE support
+    /// EDEN network will reject transactions where the maximum gas cost in ETH
+    /// is over 1.0.
+    TransactionTooExpensive,
```

Comment:
Not super important, but here I was aiming to give a prefix to errors that are tied to a specific submitter (for example `OpenEthereumTooCheapToReplace`) so we could easily recognize common errors vs submitter specific errors. If we wanted to remove Eden in the future, we wouldn't be wondering if any of the other submitters return the same error.

Reply:
So the suggestion would be renaming to `EdenTransactionTooExpensive`

Reply:
Renamed.